theme/theme.inc

```diff
@@ -19,13 +19,6 @@ function template_preprocess_islandora_basic_collection_wrapper(&$variables) {
   $page_number = (empty($_GET['page'])) ? 0 : $_GET['page'];
   $page_size = (empty($_GET['pagesize'])) ? variable_get('islandora_basic_collection_page_size', '12') : $_GET['pagesize'];
   $islandora_object = $variables['islandora_object'];
-  try {
-    $dc = $islandora_object['DC']->content;
-    $dc_object = DublinCore::importFromXMLString($dc);
-  }
-  catch (Exception $e) {
-    drupal_set_message(t('Error retrieving object %s %t', array('%s' => $islandora_object->id, '%t' => $e->getMessage())), 'error', FALSE);
-  }
   $variables['islandora_dublin_core'] = isset($dc_object) ? $dc_object : NULL;
```

Comment:
I think the issets can be removed now, and with them removed the vars become empty and the templates that reference them can be simplified.